tests/healthcheck_test.go

```diff
@@ -0,0 +1,206 @@
+				Eventually(sess).Should(Say(`HTTP Get Probe: Path="/" Port=80 HTTPHeaders=[]`))
+				Expect(err).NotTo(HaveOccurred())
+				Eventually(sess).Should(Exit(0))
+
+				sess, err = cmd.Start("deis healthchecks:list -a %s", &user, app.Name)
+				Eventually(sess).Should(Say("=== %s Healthchecks", app.Name))
+				Eventually(sess).Should(Say(`HTTP Get Probe: Path="/" Port=80 HTTPHeaders=[]`))
+				Expect(err).NotTo(HaveOccurred())
+				Eventually(sess).Should(Exit(0))
+			})
+
+			Specify("that user can set a tcpSocket readiness healthcheck", func() {
+				sess, err := cmd.Start("deis healthchecks:set readiness tcpSocket -a %s 80", &user, app.Name)
+				Eventually(sess).Should(Say("Applying readinessProbe healthcheck..."))
+				Eventually(sess, settings.MaxEventuallyTimeout).Should(Say("=== %s Healthchecks", app.Name))
+				Eventually(sess).Should(Say(`TCP Socket Probe: Port=80`))
+				Expect(err).NotTo(HaveOccurred())
+				Eventually(sess).Should(Exit(0))
+
+				sess, err = cmd.Start("deis healthchecks:list -a %s", &user, app.Name)
+				Eventually(sess).Should(Say("=== %s Healthchecks", app.Name))
+				Eventually(sess).Should(Say(`TCP Socket Probe: Port=80`))
+				Expect(err).NotTo(HaveOccurred())
+				Eventually(sess).Should(Exit(0))
+			})
+
+			Context("and has already has a healthcheck set", func() {
+
+				BeforeEach(func() {
+					sess, err := cmd.Start(`deis healthchecks:set readiness exec -a %s -- /bin/true`, &user, app.Name)
+					Eventually(sess).Should(Say("Applying readinessProbe healthcheck..."))
+					Eventually(sess, settings.MaxEventuallyTimeout).Should(Say("=== %s Healthchecks", app.Name))
+					Eventually(sess).Should(Say(`Exec Probe: Command=[/bin/true]`))
+					Expect(err).NotTo(HaveOccurred())
+					Eventually(sess).Should(Exit(0))
+				})
+
+				Specify("that user can unset that healthcheck", func() {
+					sess, err := cmd.Start("deis healthcheck:unset -a %s readiness", &user, app.Name)
+					Eventually(sess).Should(Say("Removing healthchecks..."))
+					Eventually(sess, settings.MaxEventuallyTimeout).Should(Say("=== %s Healthchecks", app.Name))
+					Eventually(sess).ShouldNot(Say(`Exec Probe: Command=[/bin/true]`))
+					Expect(err).NotTo(HaveOccurred())
+					Eventually(sess).Should(Exit(0))
+
+					sess, err = cmd.Start("deis healthchecks:list -a %s", &user, app.Name)
+					Eventually(sess).Should(Say("=== %s Healthchecks", app.Name))
+					Eventually(sess).ShouldNot(Say(`Exec Probe: Command=[/bin/true]`))
+					Expect(err).NotTo(HaveOccurred())
+					Eventually(sess).Should(Exit(0))
+				})
+			})
+		})
+	})
+
+	DescribeTable("any user can get command-line help for healthchecks", func(command string, expected string) {
+		sess, err := cmd.Start(command, nil)
+		Eventually(sess).Should(Say(expected))
+		Expect(err).NotTo(HaveOccurred())
+		Eventually(sess).Should(Exit(0))
+		// TODO: test that help output was more than five lines long
```

Comment:
sure! I've created https://github.com/deis/workflow-e2e/issues/244 to file the existing TODO in config_test.go and will update with this reference as well